src/components/atoms/icons/ArrowRight.tsx

```diff
@@ -0,0 +1,18 @@
+const DEFAULT_COLOR = "#D4ED7A";
+
+export const ArrowRight = ({ fill = DEFAULT_COLOR }) => {
```

Comment:
@eduramme let's, instead of having 4 arrow icon components, have a single one and rotate it with tailwind! Can you please remove 3 comps and rename the other one to ArrowIcon?